lib/phoenix_html/safe.ex

```diff
@@ -70,4 +70,11 @@ end
 
 defimpl Phoenix.HTML.Safe, for: Tuple do
   def to_iodata({:safe, data}), do: data
```

Comment:
Good catch. I believe we should raise a Protocol.UndefinedError, here is an example from Elixir:

https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/list/chars.ex#L28-L31

That would at least improve the error message.

I am not sure if we should do the translate_error bit though. Theoretically users should not run into this because we have released a new major phoenix_ecto version, so they would just run into this if updating phoenix_ecto explicitly.